src/components/Calculator.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Calculator.css';
+
+function Display({ value }) {
+  return <div className="display">{value}</div>;
+}
+
+function Calculator() {
+  return (
+    <div className="calculator">
+      <Display value={0} />
+      <div className="buttons">
+        <button type="button">AC</button>
+        <button type="button">+/-</button>
+        <button type="button">% </button>
+        <button type="button" className="operator">
+          ÷
+        </button>
+        <button type="button">7</button>
+        <button type="button">8</button>
+        <button type="button">9</button>
```

Comment:
- [ ] Good work up to this point but there is a project requirement you have not met yet. The calculator component should have at least 1 child component